packages/headless/src/controllers/insight/results-per-page/headless-insight-results-per-page.ts

```diff
@@ -33,6 +33,12 @@ export function buildResultsPerPage(
   return {
     ...coreController,
 
+    get state() {
```

Comment:
Does not seem necessary.

Reply:
it doesn't work without it, when we click on result per page